explorer-ui/src/components/App/App.jsx

```diff
@@ -96,8 +87,13 @@ export default function App() {
       } */}
       <Routes>
         <Route path="/" element={<Home />}/>
-        <Route path="login" element={<LoggedOutView />}/>
+        <Route path="login" element={<LoggedOutView handleLogin={handleLogin}/>}/>
         <Route path="/matches" elememt={<MatchGrid />}/>
+        <Route path="/register" element={<RegisterForm />} />
+        {isLoggedIn
+         ? <Route path="/profileView" element={<ProfileView handleCreateProfile={handleCreateProfile} profileCreated={profileCreated} profileEdited={profileEdited}/>}/>
```

Comment:
for this within the route - can do something like:
https://riptutorial.com/reactjs/example/30238/higher-order-component-that-checks-for-authentication

higher order component to redirect based on login

then can do the checks per route vs. having the ternary